spec/spec_helper_acceptance.rb

```diff
@@ -1,25 +1,25 @@
 require 'beaker-rspec'
 require 'beaker/puppet_install_helper'
 require 'beaker/module_install_helper'
+require 'beaker/testmode_switcher'
+require 'beaker/testmode_switcher/dsl'
 
 run_puppet_install_helper
 
 install_module_dependencies_on(hosts)
 
-test_name 'Installing Puppet Modules' do
-  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
-  staging = { module_name: 'puppetlabs-reboot' }
-  local = { module_name: 'reboot', source: proj_root }
+proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+staging = { module_name: 'puppetlabs-reboot' }
+local = { module_name: 'reboot', source: proj_root }
 
-  hosts.each do |host|
-    step 'Install Reboot Module Dependencies'
-    on(host, puppet('module install puppetlabs-stdlib'))
-    on(host, puppet('module install puppetlabs-registry'))
+hosts.each do |host|
+  step 'Install Reboot Module Dependencies'
```

Comment:
Do you still need these step lines if the `test_name` line was removed?

Reply:
Fixed